instance/models/mixins/openedx_config.py

```diff
@@ -425,10 +425,14 @@ def _get_configuration_variables(self):
             forum_hb_path = "lms.lib.comment_client.utils.check_forum_heartbeat"
         template["EDXAPP_LMS_ENV_EXTRA"]["HEARTBEAT_EXTENDED_CHECKS"].append(forum_hb_path)
 
-        # master has djangoapps.heartbeat installed by default now.
+        # master and newer releases djangoapps.heartbeat installed by default now.
```

Comment:
Nit: might be better to say `master and juniper release onwards has djangoapps.heartbeat installed by default`